### PR TITLE
Dr sidebar fixes

### PR DESCRIPTION
**Pull request by dretelny**
Will write a summary soon.

### CODE REVIEW

**Comment by ato1120** [comment]
So basically I added the buttons to the sidebar and in the alert above the timeline. Michael thinks we should leave both. He had some suggestions for different colors for the tasks and stuff but that will go into a separate pull request. Right now if two tasks start at the same time, the buttons/status text only appear for one of them (we will have to deal with that in a separate PR but hopefully it's not too problematic for the pilot). 

I changed the opacity of the tasks so that the worker's tasks are darker in worker view and the the tasks of other workers are lighter (it used to be the opposite). I also fixed some awareness update issues I encountered as I was coding and I added a button to the deliverables folder to each task modal

 I also removed the google drive button from the worker's sidebar although I have it still appear if the project hasn't been started and he worker hasn't logged in

So basically when testing, create a team, add a worker (or a few) and assign them to tasks, open the project in the author and worker view, start the project, start tasks, let them get delayed, paused, etc.
And just make sure everything works as it should
If there are bugs, let me know. Otherwise keep note of things that would be nice to have but not critical

**Comment by ato1120** [comment]
The big glaring issue I'm seeing right now is that you can now absolutely get away with staring your task *without* reading the task description. If you just hit start on the sidebar or alert, you never need to open the modal. 

The best solution I can think of is to do the same kind of alert box we do when you star the team. Something like "Have you read your task instructions?" and then a "Go back/cancel" button which closes the box or a "Yes, Start Task" button.

**Comment by ato1120** [comment]
When your task is completed, the buttons are still visible to "Complete" and "Take a Break" even though the text above says "You've completed all your tasks!"

**Comment by ato1120** [comment]
BUG: If I click "Pause Task" on the sidebar, click (x) on the timeline alert box, and then click "Resume Task" on the sidebar, the task resumes, the text updates and says the task is in progress, BUT the buttons will not change and will still only say "Resume Task" on the sidebar. Clicking this will do nothing. 

Again looks fine after refresh.

**Comment by ato1120** [comment]
Bug: Editing the team is really slow and struggling now when other tasks are live. I'm not sure if this or one of the most recent merges did this. It seems to work better on master, but the problem still exists there. 

Really hard to add a new task. It will keep disappearing from the timeline and sometimes firing a console error. This was while running two tasks with two different workers (a simulation of our upcoming dev tasks).
